From 6e11156f5449dda9478415493ece092a1e0301e4 Mon Sep 17 00:00:00 2001
From: John Kacur <jkacur@redhat.com>
Date: Fri, 17 May 2013 14:24:47 +0200
Subject: [PATCH 1/7] git-housekeeping: Add the .gitattributes file

By adding the .gitattributes file, git-archive can be informed not to
include files such as .gitignore and ironically .gitattributes
when creating an archive.

Signed-off-by: John Kacur <jkacur@redhat.com>
---
 .gitattributes | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 .gitattributes

diff --git a/.gitattributes b/.gitattributes
new file mode 100644
index 000000000000..596615322fb3
--- /dev/null
+++ b/.gitattributes
@@ -0,0 +1,2 @@
+.gitattributes export-ignore
+.gitignore export-ignore
-- 
1.8.3.1

